### PR TITLE
new religion yaml

### PR DESCRIPTION
**Pull request by thutch17**
Added in a new yaml for the Religion minor requirements given here: https://religion.princeton.edu/undergraduate/religion-minor.
(apologies if I did anything in the process wrong)

### CODE REVIEW

File: backend/minors/REL.yaml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        - REL 312
+        - REL 317
+        - REL 346
+        - REL 347
+        - REL 363
+        - REL 364
+- name: Research Symposium
+  max_counted: 1
+  min_needed: 1
+  explanation: |-
```
Comment (issacli-0821) [inline backend/minors/REL.yaml:166]: Add `no_req` for requirements we cannot check

File: backend/minors/REL.yaml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      At least four of these courses must be offered by the Department of Religion. One course may be 
+      offered elsewhere in the University, as long as it is approved as suitably relevant by the DUS. 
+      Courses cross-listed with REL may be considered pre-approved.
+    double_counting_allowed: false
+    pdfs_allowed: 1
+    course_list:
+    - REL ***
+  - name: Multiple areas of study
+    max_counted: 1
+    min_needed: 2
```
Comment (issacli-0821) [inline backend/minors/REL.yaml:36]: Make sure you include courses from the fall [semester](https://religion.princeton.edu/courses/undergraduate-courses-fall-2024) too!

File: backend/minors/REL.yaml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        - REL 261
+        - REL 311
+        - REL 312
+        - REL 317
+        - REL 346
+        - REL 347
+        - REL 363
+        - REL 364
+  - name: Focus on an area of study
+    max_counted: 1
```
Comment (issacli-0821) [inline backend/minors/REL.yaml:100]: We should reorganize the reqs so we only have the "Multiple Areas of Study", and make it have min_needed 2. Then make each of the area a subreq under that with min_needed = 1 and max_counted = 4 to account for the fact that we need a category that has two courses. Feel free to try something else if there's a better way to go about it

**Review by issacli-0821 — CHANGES_REQUESTED** [review]
Left some comments. Let me know if you have any questions!

**Comment by thutch17** [comment]
Should have fixed the comments now :)

File: backend/minors/REL.yaml [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  double_counting_allowed: false
+  req_list:
+  - name: REL Courses
+    max_counted: 1
+    min_needed: 5
+    explanation: |-
+      At least four of these courses must be offered by the Department of Religion. One course may be 
+      offered elsewhere in the University, as long as it is approved as suitably relevant by the DUS. 
+      Courses cross-listed with REL may be considered pre-approved.
+    double_counting_allowed: false
```
Comment (issacli-0821) [inline backend/minors/REL.yaml:30]: We prob want double_counting to be true, so courses can satisfy the 5-course req and the "Areas of Study"

**Review by issacli-0821 — CHANGES_REQUESTED** [review]
Left one more comment, and after that we should be good to merge

**Review by issacli-0821 — APPROVED** [review]
Looks good!

**Pull request merged**